smallrye-reactive-messaging-provider/src/main/java/io/smallrye/reactive/messaging/impl/InternalChannelRegistry.java

```diff
@@ -14,51 +16,50 @@
 @ApplicationScoped
 public class InternalChannelRegistry implements ChannelRegistry {
 
-    private static final String NAME_MUST_BE_SET = "'name' must be set";
     private final Map<String, List<PublisherBuilder<? extends Message<?>>>> publishers = new HashMap<>();
     private final Map<String, List<SubscriberBuilder<? extends Message<?>, Void>>> subscribers = new HashMap<>();
     private final Map<String, Emitter<?>> emitters = new HashMap<>();
 
     @Override
     public synchronized PublisherBuilder<? extends Message<?>> register(String name,
             PublisherBuilder<? extends Message<?>> stream) {
-        Objects.requireNonNull(name, NAME_MUST_BE_SET);
-        Objects.requireNonNull(stream, "'stream' must be set");
+        Objects.requireNonNull(name, msg.mustBeSet("name"));
```

Comment:
Can we avoid passing "name" and have a dedicated method?

Reply:
Done, created dedicated methods